e2e/scripts/check-installation-state.sh

```diff
@@ -34,6 +34,9 @@ main() {
     echo "ensure that installation is installed"
     wait_for_installation
     kubectl get installations --no-headers | grep -q "Installed"
+
+    echo "ensure that the admin console branding is available"
+    kubectl get cm -n kotsadm kotsadm-application-metadata
```

Comment:
Added to `check-installation-state.sh` so we can ensure we have branding during the initial install flow.